src/components/program/styles/_endorsements.scss

```diff
@@ -0,0 +1,69 @@
+.callout-wrapper {
+  .content {
+    background: #f0f2f2;
+    border: 2px solid #bfc9ca;
+    border-radius: 4px;
+    padding: 30px 48px;
+    margin-bottom: 30px;
+  }
+}
+
+ .endorsements {
+  margin: 0 auto;
+  padding-bottom: 30px;
+
+  .quote-icon-wrapper {
+    display: none;
+
+    @include media-breakpoint-up(md) {
```

Comment:
We can also use `MediaQuery` and `breakpoints` inside `jsx`. Please see [Programs.jsx](https://github.com/edx/frontend-app-learner-portal-enterprise/blob/master/src/components/program/Program.jsx) for an example